app/src/main/res/values-land/dimens.xml

```diff
@@ -253,4 +253,10 @@
   <dimen name="concept_card_explanation_text_marginTop">24dp</dimen>
   <dimen name="concept_card_explanation_text_marginEnd">64dp</dimen>
 
+  <!-- All Topics -->
+  <dimen name="all_topics_text_marginStart">72dp</dimen>
```

Comment:
rename this to `all_topics_text_margin_start`

Same applies for all values that you have introduced.

I understand that the convention that you used is part of `concept_card` but its incorrect. We should file an issue for that. Can you file an issue and mention it here?

Reply:
done changes and also created the issue #3530